app/src/interfaces/file-image/file-image.vue

```diff
@@ -7,7 +7,17 @@
 		</v-notice>
 
 		<div v-else-if="image" class="image-preview" :class="{ 'is-svg': image.type && image.type.includes('svg') }">
-			<img :src="src" alt="" role="presentation" />
+			<div v-if="imageError" class="image-error">
+				<v-icon large :name="imageError === 'illegalAssetTransformation' ? 'info_outline' : 'error_outline'" />
+				<span class="message">
+					{{
+						imageError === 'illegalAssetTransformation'
+							? t('image_errors.too_large_to_preview')
```

Comment:
We already have an "ERRORS" key under translations. Maybe we can simply use the "translateAPIError" utility for this purpose :thinking: